templates/base.html

```diff
@@ -49,6 +49,8 @@
       rel="stylesheet"
       href="{{ url_for('static', filename='css/style.css') }}"
     />
+    
+    <script type="text/javascript" src="../static/js/analysis.js"></script>
```

Comment:
Change the value or src from `../static/js/analysis.js` to `{{ url_for('static', filename='js/analysis.js') }}`